packages/template-retail-react-app/app/pages/account/payments.test.js

```diff
@@ -22,21 +26,61 @@ jest.mock('@chakra-ui/toast', () => {
 })
 
 const MockedComponent = () => {
-    const customer = useCustomer()
+    const {isRegistered} = useCustomerType()
+    const login = useShopperLoginHelper(ShopperLoginHelpers.LoginRegisteredUserB2C)
+    const customer = useCurrentCustomer()
     useEffect(() => {
-        customer.login('test@test.com', 'password')
+        if (!isRegistered) {
+            login.mutate(
+                {email: 'email@test.com', password: 'password1'},
+                {
+                    onSuccess: () => {
+                        window.history.pushState({}, 'Account', createPathWithDefaults('/account'))
+                    }
+                }
+            )
+        }
     }, [])
     return (
         <div>
-            <div>{customer.customerId}</div>
+            <div>
+                <span>Customer Id:</span>
+                {customer.customerId}
+            </div>
```

Comment:
Is this a UI change? Who requested the change? Screenshot might be helpful

Reply:
This is just a mock component that wrap the payment page, it is not important for the test

Reply:
Having customerId here to make sure the user has been signed in successfully so we can start assert for adding/removing payment